modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -48,48 +56,87 @@ If the git protocol is blocked, you can configure Git to use `https://` instead
 git config --global url."https://".insteadOf git://
 ----
 
+== Authentication
 
-== Reference
+The three ways to configure Anypoint CLI with credentials are username and password, client ID and client secret, or a bearer token.
```

Comment:
This wording in line is awkward.  I suggest something like this: 
"You can configure Anypoint CLI by using a username and password, client ID and client secret, or a bearer token." or "To configure Anypoint CLI, use a username and password, client ID, or a bearer token.

Reply:
Changed.